osm2pgsql.lua

```diff
@@ -438,41 +440,182 @@ function osm2pgsql.process_node(object)
     if object.tags.place and object.tags.wikidata == 'Q200667' then
         output_hstore['disputed_by'] = 'PK'
     end
+-- Recast Gaza Strip label as country
+    if object.tags.place and object.tags.wikidata == 'Q39760' then
+        output_hstore['place'] = 'country'
+        output_hstore['disputed_by'] = 'ID;PK;SA'
```

Comment:
But then turns it off for Indonesia, Pakistan, and Saudi Arabia?

Reply:
Ah, you pair with Israel label changes below, works for me.

You could consider instead changing to

> output_hstore['place:ID'] = 'region'
> output_hstore['place:PK'] = 'region'
> output_hstore['place:ID'] = 'region'

So the label would remain but as a region label instead of country label.

Reply:
Sure, added that